src/frameworks.sources

```diff
@@ -2235,6 +2235,7 @@ MACCATALYST_FRAMEWORKS = \
 	FileProvider \
 	GameController \
 	GameplayKit \
+	HomeKit \
```

Comment:
I think it's best to not include HomeKit at all in Mac Catalyst until the xtro tests are working properly, otherwise we could easily be adding API that's not supposed to be in HomeKit.

Reply:
https://github.com/xamarin/maccore/issues/2476#issuecomment-887017531 works around the issue

Reply:
Okay I did this workaround and made the appropriate changes. There was a clump of selectors that were ignored in iOS, tvOS, and watchOS. I assumed that these needed to be ignored in MacCatalyst as well so I moved to the common ignore, but I can keep them in the individual ignores in favor of the git blame if needed!